data/json/monsters/rodentkin.json

```diff
@@ -1,4 +1,47 @@
 [
+  {
+    "id": "mon_black_rat2",
+    "type": "MONSTER",
+    "//": "Duplicate rats created to prevent all rats from evolving into mutant rats, any monster ending in rat2 should only appear in places where mutant rats can evolve.",
```

Comment:
```suggestion
    "//": "Duplicate rats created to prevent all rats from evolving into mutant rats, any monster ending in ratkin should only appear in places where mutant rats can evolve.",
```